code/modules/clothing/outfits/ert.dm

```diff
@@ -537,3 +537,111 @@
+
+/datum/outfit/centcom/ert/marine/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+
+	if(visualsOnly)
+		return
+	var/obj/item/radio/R = H.ears
+	R.keyslot = new /obj/item/encryptionkey/heads/captain
+	R.recalculateChannels()
+
+/datum/outfit/centcom/ert/marine/security
+	name = "Marine Heavy"
+
+	id = /obj/item/card/id/advanced/centcom/ert/security
+	suit = /obj/item/clothing/suit/armor/vest/marine/security
+	suit_store = /obj/item/gun/ballistic/automatic/wt550
+	back = /obj/item/shield/riot
+	belt = /obj/item/storage/belt/military/assault/full
+	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
+	r_pocket = /obj/item/grenade/smokebomb
+	mask = /obj/item/clothing/mask/gas/sechailer
+	head = /obj/item/clothing/head/helmet/marine/security
+	l_hand = /obj/item/flashlight/seclite
+
+/datum/outfit/centcom/ert/marine/security/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+
+	if(visualsOnly)
+		return
+
+	var/obj/item/radio/R = H.ears
+	R.keyslot = new /obj/item/encryptionkey/heads/hos
+	R.recalculateChannels()
+
+/datum/outfit/centcom/ert/marine/medic
+	name = "Marine Medic"
+
+	id = /obj/item/card/id/advanced/centcom/ert/medical
+	suit = /obj/item/clothing/suit/armor/vest/marine/medic
+	suit_store = /obj/item/storage/belt/holster/detective/full/ert
+	back = /obj/item/storage/backpack/ert/medical
+	r_pocket = /obj/item/healthanalyzer
+	mask = /obj/item/food/lollipop
+	head = /obj/item/clothing/head/helmet/marine/medic
+	backpack_contents = list(
+		/obj/item/reagent_containers/hypospray/combat = 1,
+		/obj/item/storage/firstaid/regular = 1,
+		/obj/item/storage/firstaid/advanced = 1
+)
+	belt = /obj/item/storage/belt/medical/paramedic
+	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
+	l_hand = /obj/item/flashlight/seclite
+
+/datum/outfit/centcom/ert/marine/medic/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+
+	if(visualsOnly)
+		return
+
+	var/obj/item/radio/R = H.ears
+	R.keyslot = new /obj/item/encryptionkey/heads/cmo
+	R.recalculateChannels()
+
+/datum/outfit/centcom/ert/marine/engineer
+	name = "Marine Engineer"
+
+	id = /obj/item/card/id/advanced/centcom/ert/engineer
+	suit = /obj/item/clothing/suit/armor/vest/marine/engineer
+	suit_store = /obj/item/gun/ballistic/shotgun/lethal
+	mask = /obj/item/clothing/mask/cigarette/robustgold
+	head = /obj/item/clothing/head/helmet/marine/engineer
+	back = /obj/item/storage/backpack/ert/engineer
+	backpack_contents = list(
+		/obj/item/construction/rcd/loaded/upgraded = 1,
+		/obj/item/pipe_dispenser = 1,
+		/obj/item/storage/box/lethalshot = 1,
+		/obj/item/grenade/c4 = 3
+)
+	belt = /obj/item/storage/belt/utility/full/powertools
+	glasses =  /obj/item/clothing/glasses/night
+	r_pocket = /obj/item/rcd_ammo/large
+	l_hand = /obj/item/flashlight/seclite
+
+/datum/outfit/centcom/ert/marine/engineer/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+
+	if(visualsOnly)
+		return
+
+	var/obj/item/radio/R = H.ears
```

Comment:
please use better variable names

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names